src/components/MessagesList/MessagesList.vue

```diff
@@ -1106,66 +1094,35 @@ export default {
 		},
 
 		/**
-		 * @param {object} options Event options
-		 * @param {boolean} options.force Set to true, if the chat should be scrolled to the bottom even when it was not before
-		 */
-		handleScrollChatToBottomEvent(options) {
-			if ((options && options.force) || this.isChatScrolledToBottom) {
-				this.scrollToBottom()
-			}
-		},
-
-		/**
-		 * Scrolls to the bottom of the list (to show reaction to the last message).
+		 * Scrolls to the bottom of the list.
+		 * @param options {object} Options for scrolling (smooth: boolean, force: boolean)
 		 */
-		scrollToBottomIfSticky() {
-			if (this.isSticky) {
-				this.scrollToBottom()
+		scrollToBottom(options = null) {
+			if (!this.$refs.scroller) {
+				return
 			}
-		},
-
-		/**
-		 * Scrolls to the bottom of the list smoothly.
-		 */
-		smoothScrollToBottom() {
 			this.$nextTick(() => {
-				if (!this.$refs.scroller) {
-					return
-				}
-
-				if (this.isWindowVisible && (document.hasFocus() || this.isInCall)) {
-					// scrollTo is used when the user is watching
-					this.$refs.scroller.scrollTo({
-						top: this.$refs.scroller.scrollHeight,
-						behavior: 'smooth',
-					})
-					this.setChatScrolledToBottom(true)
-				} else {
+				if (!this.isWindowVisible || (!document.hasFocus() && !this.isInCall)) {
+					const firstUnreadMessageHeight = this.$refs.scroller.scrollHeight - this.$refs.scroller.scrollTop - this.$refs.scroller.offsetHeight
 					// Otherwise we jump half a message and stop autoscrolling, so the user can read up
-					if (this.$refs.scroller.scrollHeight - this.$refs.scroller.scrollTop - this.$refs.scroller.offsetHeight < 40) {
-						// Single new line from the previous author is 35px so scroll half a line
-						this.$refs.scroller.scrollTop += 10
-					} else {
-						// Single new line from the new author is 75px so scroll half an avatar
-						this.$refs.scroller.scrollTop += 40
-					}
+					// Single new line from the previous author is 35px so scroll half a line (10px)
+					// Single new line from the new author is 75px so scroll half an avatar (40px)
+					this.$refs.scroller.scrollTop += firstUnreadMessageHeight < 40 ? 10 : 40
```

Comment:
Let's apply smooth scrolling here as well, even if chat is barely visible, it still jumps